src/hyperlight_host/src/func/host_functions.rs

```diff
@@ -27,6 +28,32 @@ use crate::sandbox::{ExtraAllowedSyscall, UninitializedSandbox};
 use crate::HyperlightError::UnexpectedNoOfArguments;
 use crate::{log_then_return, new_error, Result};
 
+/// The definition of a function exposed from the host to the guest
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct HostFunctionDefinition {
```

Comment:
What do we use this for?

Reply:
When we register host functions, we save them as `HostFunctionDefinition`.